containerm/cli/cli_command_ctrs_list.go

```diff
@@ -69,6 +87,46 @@ func (cc *listCmd) setupFlags() {
 	flagSet := cc.cmd.Flags()
 	// init name flags
 	flagSet.StringVarP(&cc.config.name, "name", "n", "", "List all containers with a specific name.")
+	flagSet.BoolVarP(&cc.config.quiet, "quiet", "q", false, "List only container IDs.")
+	flagSet.StringSliceVar(&cc.config.filter, "filter", nil, "Lists only containers with the filter.")
+}
+
+func filterBy(input []string, ctrs []*types.Container) ([]*types.Container, error) {
+	var (
+		holderStatus      string
+		holderImage       string
+		convertedExitCode int = -1
+		err               error
+	)
+	filteredCtrs := []*types.Container{}
+	for _, inp := range input {
+		if strings.HasPrefix(inp, "status=") {
+			holderStatus = strings.TrimPrefix(inp, "status=")
+		} else if strings.HasPrefix(inp, "image=") {
+			holderImage = strings.TrimPrefix(inp, "image=")
+		} else if strings.HasPrefix(inp, "exitcode=") {
+			exitcodeHolder := strings.TrimPrefix(inp, "exitcode=")
+			convertedExitCode, err = strconv.Atoi(exitcodeHolder)
+			if err != nil {
+				return nil, err
+			}
+		} else {
+			return nil, fmt.Errorf("no such filter")
```

Comment:
If there is more than one filters provided, it is unclear which one is incorrect.
```suggestion
			return nil, fmt.Errorf("no filter: %s", strings.Split(inp, "=")[0])
```